app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_name, path_with_file = command.lstrip("mv").split()
+    new_file_name = path_with_file.split("/")[-1]
+    prim_path = os.getcwd()
+    new_path = "/".join(path_with_file.split("/")[:-1:])
```

Comment:
You are using the same piece twice: `path_with_file.split("/")`. Consider moving it to a variable